### DISCUSSION TITLE
Question about server settings

### DISCUSSION THREAD

**Discussion by michaelsomerville** [postalserver/postal#1716] [Help with using Postal]
Okay, I have spent about an hour with all the videos and tuts available for Postal. I am not ready to give this a go, but I have a few questions hoping the community can help me.

- Is it possible to have cpanel / cyber panel on the root ( example.com ) for the ability to have emails ( bounces, FBL, etc ) ? 
- installing postal on say postal.example.com and sending the email from email@example.com through the postal system.

Basically, I am use to using cpanel and want to know if I can have it on the server or something like it to get emails back.

**Comment by willpower232** [comment]
You could probably configure messages from whatever your cpanel uses to host emails to route via Postal but it would be annoying to get the bounce information back.

Postal works better with software that doesn't already know how to handle emails like website CMSes or other software that doesn't have mailboxes or lists.

**Reply by michaelsomerville** [reply]
Do you know anything about

workers:
  quantity: 4
  threads: 4

Read the commits on here about it, but I don't have experience with "bash" files and really got lost knowing what to edit. lol

**Reply by willpower232** [reply]
I think threads are still in use but I believe quantity is now handled by docker compose so you need to use the scale option if you want more than one, have a look at #1604

**Comment by michaelsomerville** [comment]
@willpower232  I've spent some serious time ready around this on here now. Without prior skills in these stacks, I am still struggling :) 
I've managed it get it up and running and sending, but now i've been trying to figure out how to get to the sql postal-mariadb so that I can clear the queue. To bad there wasn't a button for this.

Never used anything like that, use to cpanel and phpmysql lol

**Reply by willpower232** [reply]
It depends how you've configured it in the docker-compose file. You would need to expose port 3306 if you hadn't already and then you can use the database client of choice to access the database. The details could be the same as the ones you've entered in the config file for Postal.
